ts/App.tsx

```diff
@@ -7,13 +7,15 @@ import { PersistGate } from "redux-persist/integration/react";
 import theme from "./theme";
 
 import configureErrorHandler from "./boot/configureErrorHandler";
+import configurePushNotifications from "./boot/configurePushNotification";
 import configureStoreAndPersistor from "./boot/configureStoreAndPersistor";
 import * as config from "./config";
 import RootContainer from "./RootContainer";
 
 // Inizialize Mixpanel and configure the global js error handler
 Mixpanel.sharedInstanceWithToken(config.mixpanelToken);
 configureErrorHandler();
+configurePushNotifications();
```

Comment:
btw not sure what is the best practice here, but there's something we need to fox also for `configureErrorHandler`: both return a `Promise` and `Promise`s can't be awaited in the global scope (see [this tslint rule](https://palantir.github.io/tslint/rules/no-floating-promises/))

Reply:
actually I just noticed that `configurePushNotifications` doesn't return a `Promise`, my point stands for `configureErrorHandler`